node/mo/pages/api/links/index.ts

```diff
@@ -1,23 +1,36 @@
+import { NextApiResponse } from "next";
 import { connect } from "../../../utils/connection"
+import { getAll, openLinksCollection, withClient, withLinksCollection } from "../../../utils/db";
 import { createMethodHandler } from "../../../utils/handle"
+import { MoLinkJSON, MoLinkPutJSON } from "../../../utils/types";
 
 export default createMethodHandler({
-  async get(req, resp) {
-    const { MoLink } = await connect();
-    resp.json(await MoLink.find().sort({ n: -1 }).limit(100));
+  async get(req, resp: NextApiResponse<MoLinkJSON[]>) {
+    await withLinksCollection(async coll => {
+      const links = await getAll(coll.find().sort({ n: -1 }).limit(100));
+      const json: MoLinkJSON[] = links.map(m => ({
```

Comment:
Should we be using MDB extended JSON for encoding/decoding?

Reply:
That may work, and will get around encoding issues.